translate/rus.json

```diff
@@ -133,6 +133,7 @@
 "engine options":"настройки движка",
 "engine":"движок",
 "Enter JSON data":"Введите данные JSON",
+"Error":"Ошибка", 
```

Comment:
Okay, lines 135 and 160 are the biggest changes, I want to explain them directly to @octopulo, since I don't see the way to attach a picture for explanation of this.